src/eHealth.couchQuery/services/lucene-query-factory.js

```diff
@@ -104,9 +105,12 @@ angular
                   .then(function(response) {
                     return response.data;
                   });
-              }, initialParams);
+              }, initialParams, initialOptions);
             } else {
               initialParams.q = q;
+              angular.extend({
```

Comment:
okay, cool then. the destination variable is the first variable in `angular.extend`. as far as i can see in the doc, that is the var which is changed in place, so being it anonymous in this case you are losing the changes